.github/workflows/snyk-test.yml

```diff
@@ -0,0 +1,57 @@
+name: Snyk Scan and Report
+
+on:
+  pull_request:
+   branches: [ main ]
+  schedule:
+    - cron:  '0 4 * * *'  # run every day at midnight
+
+permissions:
+  id-token: write
+
+jobs:
+  snyk_run:
+    name: Snyk Run (for PR and push)
+    runs-on: ubuntu-latest
+    if: github.event_name == 'pull_request'
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+    
+      - name: Install Snyk and Run Snyk test
+        run: |
+          npm install -g snyk
+          snyk test --all-projects --json > snyk_output.txt || true
+        env:
+          SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+    
+
+  snyk_nightly_run:  
+      name: Snyk Nightly Run (for nightly cron with JIRA)
+      runs-on: ubuntu-latest
+      if: github.event_name == 'schedule'
+      steps:
+        - name: Check out repository
+          uses: actions/checkout@v2
+
+        - name: Install Snyk and Run Snyk test
+          run: |
+            npm install -g snyk
+            snyk test --all-projects --json > snyk_output.txt || true
+          env:
+            SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}     
+    
+        - name: use the custom github  action to parse Snyk output
+          uses: Enterprise-CMCS/macfc-security-scan-report@v1.0.2
+          with:
+            jira-username: ${{ secrets.JIRA_SERVICE_USERNAME }}
+            jira-token: ${{ secrets.JIRA_SERVICE_USER_TOKEN }}
+            jira-host: 'qmacbis.atlassian.net'
+            jira-project-key: 'MDCT'
+            jira-issue-type: 'Task'
+            jira-custom-field-key-value: '{ "customfield_10007" : "MDCT-2280"  }'
```

Comment:
Is it possible to add the MCR application tag here?

`jira-custom-fields: '{ "customfield_14154" : [{"id": "16958", "value": "MFP"}] }'`

Reply:
the tag has been added